compiler/plugins/input/StableHLO/PluginRegistration.cpp

```diff
@@ -115,6 +117,7 @@ struct StableHLOSession
   void populateCustomInputConversionTypes(StringSet<> &typeMnemonics) override {
     typeMnemonics.insert("stablehlo");
     typeMnemonics.insert("stablehlo_xla");
+    typeMnemonics.insert("vhlo");
   }
 
   void populateDetectedCustomInputConversionTypes(
```

Comment:
This still needs to be updated to detect vhlo. After vhlo is detected here, the "auto detect" changes in iree/compiler/Pipelines/Pipelinces.cpp won't be needed to get the desired behavior.

Reply:
fixed.